### ISSUE TITLE
Hover handlers don't work in Vega-Lite

### ISSUE THREAD

**Issue by domoritz** [vega/react-vega#48]

**Comment by hungngo97** [comment]
What do you mean is doesn't work? I ran the storybook example and hovers on some of the bars and it updates the component state

**Comment by domoritz** [comment]
Hmm, didn't work for me. Let me try again.

**Comment by domoritz** [comment]
Definitely doesn't work for me. 

![editor](https://user-images.githubusercontent.com/589034/61115280-651b9e00-a492-11e9-9267-933e6067525e.gif)

**Comment by hungngo97** [comment]
I did yarn install && yarn build && yarn storybook from a fresh new cloned repo

**Comment by domoritz** [comment]
And you then see the actions being logged like in Vega?

**Comment by kristw** [comment]
@domoritz I'm not sure how to fix this one as `vega-lite` doesn't issue signal `hover`. The tooltip does work tho.

**Comment by domoritz** [comment]
Vega-Embed doesn't enable hover handling by default for Vega-Lite. See https://github.com/vega/vega-embed/blob/5042fe105f7ede9500f3cbc4211bfddbd1bc13f8/src/embed.ts#L297. Maybe that's the reason this doesn't work.

**Comment by eeyshen** [comment]
Hover handler do not work in vega-lite for me, too. Is there any way to get it done? Thanks.
